Web Development/Password Strength Checker/stylesheet.css

```diff
@@ -0,0 +1,74 @@
+*,
```

Comment:
u don't need to specify *, * :before, * :after. * means "all" it basically removes the default padding and margin from the document. So only * is needed. As by default no content is present.